Makefile

```diff
@@ -37,7 +37,11 @@ CMAKE_ARGS ?=
 VERBOSE ?=
 FORCE_CLANG_FORMAT ?=
 
-pyextsuffix := $(shell python3-config --extension-suffix)
+WHICH_PYTHON := $(shell which python)
+REALPATH_PYTHON := $(shell realpath $(WHICH_PYTHON))
```

Comment:
Please use make function while we can: https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html .  It runs faster than shell.

Reply:
Let me check. Thanks for the tip!